README.md

```diff
@@ -1,3 +1,35 @@
 # SnowMan
 
-This is a console Swift application. It plays like Hangman but without the gore.
+Snowman is a Swift console application. The object of the game is to guess the given word or phrase before the snowman "melts". With each incorrect guess, the snowman will lose one of its snowman features until he is but a puddle. 
+
+## Gameplay
+
+To play the game... TBD
+
+## Running the Application
+
+SnowMan is built with Apple Swift (version 2.2). It encorporates [Swift Package Manager](https://github.com/apple/swift-package-manager) for source code distribution.
```

Comment:
> SnowMan is built with Apple Swift (version 2.2)
Might want to mention the snapshot version too, incase future snapshots start throwing deprecation warnings (like they did on swiftest).